packages/core/src/Provider/HandleProvider/index.ts

```diff
@@ -0,0 +1,47 @@
+import { HandleInfo, HandleProvider, ResolveHandlesArgs } from './types';
+import { HealthCheckResponse, NetworkInfoProvider, ProviderError, ProviderFailure } from '../..';
+import { IHandle } from '@koralabs/handles-public-api-interfaces';
+import { toHandleInfo } from './utils';
+import axios, { AxiosInstance } from 'axios';
+
+export interface KoraLabsHandleProviderDeps {
+  serverUrl: string;
+  networkInfoProvider: NetworkInfoProvider;
+}
+
+export class KoraLabsHandleProvider implements HandleProvider {
```

Comment:
I believe `wallet` shouldn't be the one because it can be used by any other application than a wallet? so that would leave `cardano-services` @rhyslbw ?

Reply:
Ok, I will move it to `wallet`.

Reply:
when moving the provider to `wallet`, I find that the addresses I need from `core` aren't exported, and only used in `core`. It's ok to make them exported, or is it a particular reason they aren't?

`import { ByronAddress, EnterpriseAddress, PaymentAddress, PointerAddress } from '@cardano-sdk/core/src/Cardano/Address'; `